src/test/java/org/cactoos/iterable/FilteredTest.java

```diff
@@ -130,12 +133,24 @@ public void withItemsNotEmpty() {
     public void withoutItemsIsEmpty() {
         new Assertion<>(
             "Must be empty without items",
-            new Filtered<String>(
+            new Filtered<>(
                 input -> input.length() > 16,
                 new IterableOf<>("third", "fourth")
             ),
             new IsEmptyIterable<>()
         ).affirm();
     }
 
+    @Test
+    public void filtersWithTwoFuncsCombined() throws Exception {
```

Comment:
@andreoss I believe the name of the test is incorrect?

Reply:
@victornoel Done